scripts/ovpsim_log_to_trace_csv.py

```diff
@@ -27,34 +27,35 @@
 from riscv_trace_csv import *
 
 try:
-    from ovpsim_log_to_trace_csv_vectors import *
+  from ovpsim_log_to_trace_csv_vectors import *
 except:
-    def assign_operand_vector(a,b,c,d):
-        """ stub version when no vector processing included """
-        logging.info("No OVPsim vector instruction processing included")
-        if stop_on_first_error:
-            sys.exit(-1)
-    def is_an_extension_instruction(instr):
-        if 'v' in instr[0]:
-            return True
-        return False
+  def assign_operand_vector(a,b,c,d):
```

Comment:
I am not sure why you have parameters `a,b,c,d`?

Reply:
It seems like this is a stub function until OVPsim vector instruction processing is added. @simond-imperas is this accurate?

Reply:
I think this needs to refactored later, doesn't have to be resolved in this PR.